modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectDefinitionLocalServiceImpl.java

```diff
@@ -740,7 +740,10 @@ private ObjectDefinition _addObjectDefinition(
 		objectDefinition.setUserName(user.getFullName());
 		objectDefinition.setActive(system);
 
-		objectDefinition.setEnableCategorization(!system);
+		objectDefinition.setEnableCategorization(
+			!system &&
+			StringUtil.equals(
```

Comment:
Sorry, @brianchandotcom. I believe this line is wrong. I will test again and send the changes